README.md

```diff
@@ -79,17 +115,40 @@ None
   roles:
     - ansible-role-monit
   vars:
+    monit_config: |
+      # disable delay during kittchen test because monit does not listen immediately
+      set daemon 10
+        with start delay 0
+      set httpd port 2812
+        use address 127.0.0.1
+        allow 127.0.0.1
+      set logfile syslog facility log_daemon
+
+    extra_user: "{% if ansible_os_family == 'FreeBSD' %}www{% elif ansible_os_family == 'OpenBSD' %}www{% elif ansible_os_family == 'Debian' %}www-data{% elif ansible_os_family == 'RedHat' %}ftp{% endif %}"
+    extra_group: "{% if ansible_os_family == 'FreeBSD' %}www{% elif ansible_os_family == 'OpenBSD' %}www{% elif ansible_os_family == 'Debian' %}www-data{% elif ansible_os_family == 'RedHat' %}ftp{% endif %}"
+    monit_conf_extra_include_directories:
+      - path: /usr/local/project/config/monit
+        state: enabled
+        mode: 750
```

Comment:
"mode", "owner" and "group" are not required as the directory is managed by the other.

Reply:
this has not been fixed.
in addition, "extra_user" and "extra_group" are not used.